pkg/types/rekord/rekord_schema.json

```diff
@@ -0,0 +1,12 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://rekor.dev/types/rekord/rekord_schema.json",
+    "title": "Rekor Schema",
+    "description": "Schema for Rekord objects",
+    "type": "object",
+    "oneOf": [
```

Comment:
the `oneOf` semantics allow for different versions of the type to be supported concurrently